README.md

```diff
@@ -115,6 +115,7 @@ Additional documentation can be found here: https://github.com/svenfuchs/i18n/wi
 * [Stephan Soller](http://www.arkanis-development.de)
 * [Saimon Moore](http://saimonmoore.net)
 * [Matt Aimonetti](https://matt.aimonetti.net/)
+* [Francisco Viramontes](https://twitter.com/kidpollo)
```

Comment:
Please remove this line. We do not add PR authors to this list. We have our list of contributors underneath this, but it's just a link to https://github.com/svenfuchs/i18n/graphs/contributors.